_includes/github_team_members.html

```diff
@@ -35,12 +35,15 @@ <h6 class="mb-3">{{  t.project_leader }} </h6>
 		<div class="carousel-footer mt-auto d-flex">
 
 			<ol class="carousel-indicators align-self-center">
-				{% assign indicators = team_members.size | modulo: 3 %}
-				{% assign checkIndicators = indicators | times: 3 %}
+        {% assign indicators = team_members.size | modulo: 3 %}
```

Comment:
Hello @UchihaCFC, thanks for the PR!

I'm not sure about this. Can't we just do `{% assign indicators = team_members.size | divided_by: 3.0 | ceil %}` and simplify the whole thing?

Reply:
You are right, it is much cleaner! In this type of templates I did not know that there was \`dividev\_by\`, thanks @bfabio, I will be really useful in the future :).

I uploaded one more commit with this change after testing that everything worked ok.